frontend/app/src/store/exchanges/index.ts

```diff
@@ -5,7 +5,6 @@ import {
   type ExchangeSetupPayload,
   type SupportedExchange
 } from '@/types/exchanges';
-import { assert } from '@/utils/assertions';
 import { useExchangeBalancesStore } from '@/store/balances/exchanges';
 import { useSessionSettingsStore } from '@/store/settings/session';
```

Comment:
Can you check in general if some of the store/composables imports can be also removed?

Reply:
yeah, removed, with some from src/composables as well